smurfs/src/view/SmurfListView.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { getSmurfs } from '../actions';
+import SmurfFormView from '../view/SmurfFormView.js';
+
+import '../index.css';
+
+
+class SmurfListView extends React.Component {
+    componentDidMount() {
+        this.props.getSmurfs();
+    }
+
+    render() {
+        return (
+            <div className="smurfListBox">
+                {this.props.grabbingSmurfs ? <h2>Grabbing as many Smurfs as we can...</h2> : null}
+                { !this.props.grabbingSmurfs && this.props.smurfs.length ? 
+                <ul>
+                    {this.props.smurfs.map(smurf =>
+                        <li key={smurf.name} className="smurfInfoBox">
```

Comment:
Just in case you have multiple elements with classNames putting that prop first will make it easier to find and change them. The key is super important though and putting it first also makes sense. _I forget to add my keys all the time_ 🙂 
```suggestion
                        <li className="smurfInfoBox" key={smurf.name} >
```